src/main/kotlin/simplerag/ragback/domain/document/service/DataFileService.kt

```
@@ -60,39 +58,37 @@ class DataFileService(
             uploadedUrls += fileUrl
 
             val dataFile = try {
-                dataFileRepository.save(DataFile(meta.title, type, sizeBytes, sha256, fileUrl))
+                dataFileRepository.save(DataFile.from(meta.title, type, sizeBytes, sha256, fileUrl))
             } catch (ex: DataIntegrityViolationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)